### ISSUE TITLE
Using hotkey "T" will select Transform Tool ignoring the current keymap where its disabled.

### ISSUE THREAD

**Issue by AlexanderSDiff** [AIGODLIKE/gesture_helper#12]
Using hotkey "T" for newly created gesture menu will switch tools to "Transform Tool". Transform tool currently did not binded to any hotkey in blender, so if i uncheck those gesture menu binded to T - transform tool would not be setted again after pressing T key.

I believe its not how it supposed to work and probably a bug in addon or blender?

**Comment by xmx-emm** [comment]
![image](https://github.com/user-attachments/assets/501e644c-e370-4bfe-b9bc-3a8bd37702c3)
Need to add keymap "Object Mode"

**Comment by AlexanderSDiff** [comment]
It doesnt help unfortunately. Behaviour is still the same: press and hold "T" - will open gesture pie, but short pressing of "T" will switch tool to "transform tool". 
I can record video if it will clarify the situation.

**Comment by xmx-emm** [comment]
What kind of operation do you want to achieve

T key only works if Keymap is Blender27x

**Comment by xmx-emm** [comment]
I need to look at the console if it's not Blender27x.
![image](https://github.com/user-attachments/assets/b8d16827-d5c3-420f-a4b8-edd40ba5864f)

**Comment by AlexanderSDiff** [comment]
Here it is.
![scr](https://github.com/user-attachments/assets/b147b579-6423-4358-a8c9-d3b266c00b11)

Here's you can see what T key doesnt used for "Set Tool by Name" at all, gesture set to "Object Mode" but it doesnt change anything. Each time i shortly press T - i will select transform tool. 
![scr2](https://github.com/user-attachments/assets/7b582414-1838-4d34-9883-864258c6e4f3)

Im pretty sure what T was used for transform tool in Industry Compatible keymap, which i was used as a default for making my own.

> What kind of operation do you want to achieve
I want to remove such behaviour, because if i by mistake press t too shortly - it will change the current active tool. 
Basically it works the same way with any hotkeys which are default to original keymap which was used, but disabled in current one.
I.e. my keymap are based on industry standart keymap. In this keymap Shift+RMB are enabling 3d cursor which you can drag. If i will try to use any gestures binded to Shift+RMB - the behaviour will be the same, i.e. if i press shift _ rmb and wait until gesture will appear - everything will be okay, but if those sequence of pressing shift + RMB will be too short - it will enable 3d cursor drag, even if this hotkey are disabled in keymap.

**Comment by xmx-emm** [comment]
![image](https://github.com/user-attachments/assets/0c8f2ef0-4423-46bd-8e42-5e90ce496f3d)
Might want to take a look here.
This should happen because there should be the same trigger inside this keymap

Keymaps shouldn't be able to pass_through when disabled, I'll fix that

shift + RMB is handled separately because it needs to be compatible with cursor operations.

Sorry for the long response time as github messages aren't real time

**Comment by xmx-emm** [comment]
I've just pushed the fix, download it from the master branch

**Comment by AlexanderSDiff** [comment]
Its worked now! Thank you!

**Comment by AlexanderSDiff** [comment]
Hello again.
I was needed to reinstall blender, and unfortunately with installing the same gesture helper zip, im again stuck with a same problem.
As a default keymap i choose an "Indistry Compatible".
I untick each and every Set Tool By Name binded to T by default.
So now i can press T in any mode, object or edit and it will do nothing.
But as soon as i will import my old gestures or create a new one and set hotkey to T, then each time i press T too shortly, or simply miss the button in gesture helper - it will enable Transform Tool.

Thats what console show me:
invoke GESTURE_OT_operator      modal   PRESS   T       prev T RELEASE
else    3D View Generic []
        call wm.tool_set_by_id  {'name': 'builtin.transform', 'cycle': True}    {'FINISHED'}

Sorry for video quality, youtube upload it as a shorts:
https://www.youtube.com/watch?v=uMNlfUZ86sk&ab_channel=SoundDifferent

**Comment by xmx-emm** [comment]
It's strange, if there's no valid counterpart operator it shouldn't search for the

Export keymap, try to reproduce it over here
![image](https://github.com/user-attachments/assets/a83da86e-2e76-4fa5-a4d4-525807e4ceb6)

**Comment by xmx-emm** [comment]
Maybe I should change the way the keymap is passed through

**Comment by xmx-emm** [comment]
Just pushed
Try setting the preference to keymaps
from gesture set keymaps find key
![image](https://github.com/user-attachments/assets/27391d33-651f-4d94-8516-77ac7076be4f)

**Comment by xmx-emm** [comment]
The new version has been released. Please try modifying the options to see if this issue still exists. If it is not resolved, please reopen the topic

https://github.com/AIGODLIKE/gesture_helper/releases/tag/v2.0.8
